# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mol/linux-2.6.10.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mol-0.9.70/src/kmod/Linux/alloc.h	2004-03-27 12:04:35.000000000 +0100
+++ mol-0.9.70-fixed/src/kmod/Linux/alloc.h	2005-02-20 19:04:24.471974256 +0100
@@ -23,6 +23,7 @@
 #include <asm/uaccess.h>
 
 #ifdef LINUX_26
+#include <asm/io.h>
 #include <asm/cacheflush.h>
 #endif
 
